docker/Dockerfile.celery.template

```diff
@@ -3,6 +3,7 @@ FROM balenalib/%%BALENA_MACHINE_NAME%%-debian:jessie
 RUN apt-get update && \
     apt-get -y install \
         build-essential \
+        cec-utils \
```

Comment:
Could you add the dependency to `ansible/roles/system/tasks/main.yml`?
`docker/Dockerfile.celery.template` only for balenaCloud